components/ReportButton.tsx

```diff
@@ -0,0 +1,40 @@
+import { Contact } from "../lib/provinces";
+
+import { ExclamationCircleIcon } from "@heroicons/react/outline";
+
+type ReportButtonProps = {
+  contact: Contact;
+  provinceName: string;
+};
+
+const ReportButton = (props: ReportButtonProps) => {
```

Comment:
```suggestion
export const ReportButton = (props: ReportButtonProps) => {
```

Please use named export to guarantee to autocomplete in VS Code editor. 😁